WebRoot/public/extuserprov.jsp

```diff
@@ -147,5 +151,10 @@
 		return !isError;
 	}
 </script>
+<% if (request.getAttribute("errorCode") != null) { %>
+	<script>
+		document.getElementById("ZLoginErrorPanel").style.display = "block";
+	</script>
+<% } %>
```

Comment:
should we also have else to this? 
if errorCode is null then we should set the element to display none.